tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -359,14 +359,17 @@ def _wait_for_toggle_ready(self, by, locator):
         Wait(self.marionette).until(expected.element_enabled(checkbox))
 
     # this method is a copy of go_back() method in regions/keyboard.py
-    def return_to_prev_menu(self, parent_view, gaia_header=True):
+    def return_to_prev_menu(self, parent_view, exit_view, gaia_header=True):
 
         # TODO: remove tap with coordinates after Bug 1061698 is fixed
         if gaia_header:
             _header_locator = self._header_locator
```

Comment:
Is this still an option that is being run, actually? Otherwise you can remove it.